scripts/calculate-advisors-rewards.py

```diff
@@ -0,0 +1,70 @@
+"""Usage: calculate-advisors-rewards.py --seed=STRING --proposals=FILE --advisors=FILE --total-incentive=INT [--output=FILE]
+
+Options:
+    --seed <string>  seed used for random selection of reviews
+    --proposals <file> file containing the reviews for each proposal
+    --advisors <file> file containing info for each advisor
+    --total-incentive <int> the amount of funding (in $) allocated to community advisors
+    --output <file> [ default: output.xlsx ]
+"""
+
+from docopt import docopt
+import random
+import pandas
+import math
+from collections import Counter
+
+NUM_WINNERS_PER_PROPOSAL = 3
+
+arguments = docopt(__doc__)
+proposals_file = pandas.read_excel(arguments["--proposals"], sheet_name=[0, 1])
```

Comment:
I'd rather avoid going through the proprietary Excel format if we can do it with simple CSV. Not sure if `ffill` works the same way with a CSV export, though.